02-express-tutorial/app.js

```diff
@@ -1,82 +1,96 @@
-// app.use
-// app.all
-// app.listen
-
-const express = require('express');
+const express = require("express");
 const app = express();
 const { products } = require("./data");
 
-// app.use to pull in static index.html page
-app.use(express.static('./public'));
+// // app.use to pull in static index.html page
+// app.use(express.static("./public"));
+app.use(express.static('./methods-public'))
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
 
+// middleware logger
+const logger = (req, res, next) => {
```

Comment:
if it's defined in the logger file you can import that rather than redefining here